.github/workflows/build.yml

```diff
@@ -24,12 +24,20 @@ jobs:
           go-version: 1.16
       - name: Checkout code
         uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
       - name: yaml-lint
         uses: ibiqlik/action-yamllint@v3
       - name: Run unit tests
         run: make tests
       - name: Upload code coverage
-        uses: codecov/codecov-action@v1
+        uses: codecov/codecov-action@v2.0.2
```

Comment:
better use major release so we won't have to update each minor release

Reply:
```suggestion
        uses: codecov/codecov-action@v2
```